08LessonEight/in-class-project/css/homework-style.css

```diff
@@ -63,12 +63,21 @@ h3{
 }
 .footer{
   margin: 50px auto;
-  width: 45%;
+  width: 50%;
+  border: 1px solid;
+  border-color: whitesmoke;
+  padding: 10px 49px;
 }
 .footerbox{
   float: left;
-  text-align: center;
 }
-.footerbox ul{
-  text-decoration: none;
+.footerbox1{
```

Comment:
What the difference between `.footerbox1` and `.footerbox2`?
If there is none, then you are repeating two classes that do the same thing.